api/webhooks/index.md

```diff
@@ -23,10 +23,63 @@ Webhooks allow you to specify a URL to which you would like Cockpit to post data
 
 All data will be send as _application/json_.
 
+### Using POST-data in the webhook destination script
 
-### Useful events
+Some hooks send data along with the hook (like some of the collection event hooks which send the fresh new collection's name and confifuration).
 
+The data is posted to the webhook destinatin script in RAW-Post format. So the global var _**$\_POST**_ won't contain any data.
```

Comment:
another typo "destinatin"

what the heck did I type here?! Should I cancel the PR in order to correct typos or would you instead correct them?

Reply:
Just do a new commit in you forked branch and it will be added to the PR.